src/components/Home/BookList/Book.js

```diff
@@ -1,6 +1,6 @@
 import './Book.css';
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../../../redux/books/booksSlice';
+import { removeBook, deleteBook } from '../../../redux/books/booksSlice';
 // eslint-disable-next-line
 import PropTypes from 'prop-types';
```

Comment:
- [ ] For best practices, clean code and as required for the project, kindly make sure you fix all your linters error instead of disabling them